package.json

```diff
@@ -71,9 +71,12 @@
   "repository": "forcedotcom/salesforcedx-vscode",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "node": "12.22.9",
+    "node": "^14.0.0",
```

Comment:
We can still use `^14.16.0` as we are using `14.16.0` in other places and that would ensure that atleast 14.16.0 is the minimum version.